runbotics/runbotics-orchestrator-ui/src/src-app/translations/pl/process/process.json

```diff
@@ -311,9 +311,11 @@
     "Process.Details.Modeler.Actions.CloudExcel.SetCell.Label": "Dodaj zawartość do komórki",
     "Process.Details.Modeler.Actions.SharePointExcel.SetCell.Content": "Zawartość komórki",
     "Process.Details.Modeler.Actions.SharePointExcel.SetCell.Cell": "Adres komórki",
-    "Process.Details.Modeler.Actions.CloudExcel.UpdateRange.Label": "Aktualizuj zakres komórek",
-    "Process.Details.Modeler.Actions.SharePointExcel.UpdateRange.Cell": "Zakres komórek",
-    "Process.Details.Modeler.Actions.SharePointExcel.UpdateRange.Values": "Wartości [zmienna]",
+    "Process.Details.Modeler.Actions.CloudExcel.SetCells.Label": "Ustaw komórki",
+    "Process.Details.Modeler.Actions.SharePointExcel.SetCells.StartCell": "Komórka początkowa",
+    "Process.Details.Modeler.Actions.SharePointExcel.SetCells.StartCell.Info": "Adres komórki początkowej (np. A3)",
```

Comment:
in desktop we're passing examples after a comma:
"Adres komórki początkowej, np. A3"